client-tools/Shell/V8ClientConnection.cpp

```diff
@@ -173,6 +173,9 @@ std::shared_ptr<fu::Connection> V8ClientConnection::createConnection(
 
       if (!res) {
         setCustomError(500, "unable to create connection");
+        LOG_TOPIC("9daaa", DEBUG, arangodb::Logger::HTTPCLIENT)
+            << "Connection attempt to endpoint '" << _client.endpoint()
+            << "' failed: " << res->statusCode();
```

Comment:
If `!res` applies here, doesn't it mean `res->statusCode()` would dereference a nullptr?